pertpy/tools/_mixscape.py

```diff
@@ -0,0 +1,225 @@
+from __future__ import annotations
+
+import numpy as np
+import pandas as pd
+import scanpy as sc
+from anndata import AnnData
+from scipy import sparse
+from sklearn.mixture import GaussianMixture
+
+
+def _define_normal_mixscape(X: np.ndarray | sparse.spmatrix | pd.DataFrame | None) -> list[float]:  # noqa: N803
+    """Calculates the mean and standard deviation of a matrix.
+
+    Args:
+        X: The matrix to calculate the properties for.
+
+    Returns:
+        Mean and standard deviation of the matrix.
+    """
+    mu = X.mean()
+    sd = X.std()
+
+    return [mu, sd]
+
+
+def mixscape(
```

Comment:
The function has a few unused arguments so far. We should ensure that the final implementation uses all of them.